docs/docs/contactus.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: docs
+title: Contact Us
+---
+
+
+# Contact Us
+Want to learn more from the Refocus team? Contact us at: cxtools at salesforce.com
```

Comment:
I'm sure you're not using an actual email address for a reason... but why?

Also, we don't really want folks emailing us for bugs and feature requests--we want them to create Issues... that's why I had that stuff in the example I sent you.